src/App.js

```diff
@@ -1,24 +1,41 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  NavLink,
+} from 'react-router-dom';
+import AddItem from './components/AddItem';
+import ItemList from './components/ItemList';
+
+// const activeLinks = () => {
+//   const [activeLink, setActiveLink] = useState(null);
+//   const changeFont = (e, index) => {
+//     console.log({ e });
+//     setActiveLink(index);
+// //   };
+
+//   return
+// }
```

Comment:
please delete or add comments if saving for later.

Reply:
Thanks for the reminder, all fixed!